.github/workflows/ci.yml

```
@@ -1,7 +1,7 @@
 name: CI
 
 on:
-  pull_request:
+  pull_request: read
```

actionlint (1.7.12)

[error] "pull_request" section is scalar node but mapping node is expected

(syntax-check)